src/mail-app/mail/view/MailView.ts

```diff
@@ -298,6 +298,7 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 			mailModel: mailLocator.mailModel,
 			mails: this.mailViewModel.listModel?.getSelectedAsArray() ?? [],
 			selectNone: () => this.mailViewModel.listModel?.selectNone(),
+			actionApplyMails: () => this.mailViewModel.getActionableMails(this.mailViewModel.listModel?.getSelectedAsArray() ?? []),
```

Comment:
we did discuss loading all mails in all of those conversations but I don't remember what we decided for, sorry

Reply:
I now realized, this doesn't even return all loaded items so far, only the latest ones, so it's even worse than I thought.